### DISCUSSION TITLE
Can I obtain data describing energies of particles passing through some regions?

### DISCUSSION THREAD

**Discussion by jburz2001** [nrc-cnrc/EGSnrc#1148] [Questions]
Hello. Is there a way to output data showing how many photons, electrons, and positrons pass through particular regions along with what their energies when they were located at these regions? I have a geometry where electrons strike an anode to produce photons then these photons travel through a copper window, where this copper serves as a high-pass filter to nullify low-energy photons. However, I want to analyze this filter's performance.

In the dosimetry in a region per particle I see Edep but I'm also curious about the energies that were not deposited (those that pass through the high-pass filter).

**Comment by rtownson** [comment]
I see that you've already closed this, but just for anyone reading in the future - a good strategy would be to score a phase-space of particles crossing a boundary. Then you can analyze the particles in the phase-space.
